apps/web/src/server/applications/case/s51/applications-s51.controller.js

```diff
@@ -73,39 +72,10 @@ export async function viewApplicationsCaseS51Item({ params, query }, response) {
 
 	const s51Advice = await getS51Advice(caseId, Number(adviceId));
 
-	const s51Files = await (async () => {
```

Comment:
don't you need this block anymore?

Reply:
nope, attachments are now a field in the single advise API